src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
```

Comment:
The interface name `AreaCalculator` is a verb phrase, which is not recommended for class/interface names. Consider renaming it to something that represents an entity or a concept, such as `AreaComputable` or `AreaProvider`.